### DISCUSSION TITLE
Run a function when "Confirm" button is pressed

### DISCUSSION THREAD

**Discussion by Anan-Saadi** [fluttercandies/flutter_wechat_assets_picker#238] [Q&A] [answered]
I was trying to push selected images to firebase but I found it difficult to handle this functionality after getting the list of `AssetEntity`s, is there a way to implement a method inside "onPressed", even if it requires forking the repo and adding it myself ?

**Comment by AlexV525** [comment, marked as answer]
await pickAssets.
